SensusService/Probes/User/ScriptRunner.cs

```diff
@@ -106,6 +107,19 @@ public bool Enabled
             }
         }
 
+        [OnOffUiProperty("Allow Cancel:", true, 2)]
```

Comment:
The order index should be sequential. It currently is equal to the order index of `Enabled` (2).